### ISSUE TITLE
Create Demos of Different Solutions

### ISSUE THREAD

**Issue by RobLoach** [kalamuna/kalastatic#306]
- Pattern Builder
- KSS + Twig
- Drupal Lab

**Comment by derekderaps** [comment]
@RobLoach @madeofpeople @soniktrooth Can we motivate a first step of moving Kalastatic from Swig to Twig.js (via KSS-node 3.0.0-beta)? This would provide a big win on Dent and GBZ3 by allowing us to re-use component templates.

**Comment by madeofpeople** [comment]
That only takes care of the style guide but not the prototyping.

On May 30, 2016 10:06:39 PM EDT, Derek DeRaps <notifications@github.com> wrote:
>@RobLoach @madeofpeople @soniktrooth Can we motivate a first step of
>moving Kalastatic from Swig to Twig.js (via KSS-node 3.0.0-beta)? This
>would provide a big win on Dent and GBZ3 by allowing us to re-use
>component templates.
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/kalamuna/kalastatic/issues/306#issuecomment-222577282

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by derekderaps** [comment]
Correcto. I thought you were on board with nixing the prototype on Dent and GBZ3. My plan was styleguide + implementation, no prototype.

**Comment by madeofpeople** [comment]
I just dnt see nixing the prototype from kalastatic as something that we should be discussing in the kalastatic repo.

Can we avoid this complexity on current projects, sure.

Now changing the codebase in a way that effectively “unsupports” static sites and prototypes (eg. breaks current functionality) is something I think would take more care/discussion and planning.

**Comment by madeofpeople** [comment]
At the same time, it does get us up and running “in the present” so I fully support it. ;)

**Comment by soniktrooth** [comment]
the prototype and the style guide aren't sharing templates atm anyway so if there is a 'twig for the style guide' solution then that could be cool. Is that what you're proposing @derekderaps ?

**Comment by derekderaps** [comment]
@Josh, yup! Prioritizing the twig switch for the sake of agency work.

**Comment by soniktrooth** [comment]
sounds like a step in the right direction to me.

**Comment by madeofpeople** [comment]
Do eeeeeeeet!

**Comment by RobLoach** [comment]
#92 is the switch over to Twig, which is a good one to read through. Outlined some of the steps required for the Twig/KSS concept there.

**Comment by RobLoach** [comment]
D8 https://www.drupal.org/project/lightning

Panels + Content Editing fixes, a few other things.

**Comment by RobLoach** [comment]
D8, Bootstrap, Styleguide (Pattern Lab or KSS).
